File: fstcomp/examples/GraphJava/BasicGraph/Graph/Edge.java
// $Header: /home/apel/cvs/fstcomp/examples/GraphJava/BasicGraph/Graph/Attic/Edge.java,v 1.3 2010-02-10 11:58:17 apel Exp $
/**
 * Please complete these missing tags
 * @author
 * @rref
 * @copyright
 * @concurrency
 * @see
 */
package Graph;

class Edge
{
	
    Node a;
    Node b;
    /**
     * Please complete the missing tags for Edge
     * @param
     * @return
     * @throws
     * @pre
     * @post
     */
    Edge( Node _a, Node _b )
    {
        a = _a;
        b = _b;
    }
    /**
     * Please complete the missing tags for print
     * @param
     * @return
     * @throws
     * @pre
     * @post
     */
    void print()
    {
        System.out.print( "edge (" );
        a.print();
        System.out.print( ", " );
        b.print();
        System.out.print( ") " );
    }
}

File: fstmerge/examples/Graph/BasicGraph/Edge.cs
class Edge {
	Node a, b;
	public Edge(Node _a, Node _b) { a = _a; b = _b; }
    public void Print() {
        System.Console.Out.Write("edge ("); 
        a.Print(); 
        System.Console.Out.Write(", ");
        b.Print();
        System.Console.Out.Write(") "); 
    }
}